cmd/earth/main.go

```diff
@@ -1172,6 +1172,30 @@ func (app *earthApp) actionRegister(c *cli.Context) error {
 		pword = string(enteredPassword)
 	}
 
+	var interactiveTermsConditions bool
+	if !app.termsConditions {
+		rawAccept := promptInput("Do you accept the Earthly Terms & Conditions? [y/N]")
+		if rawAccept == "" {
+			rawAccept = "n"
+		}
+		accept := strings.ToLower(rawAccept)[0]
+
+		interactiveTermsConditions = accept == 'y'
+	}
+	termsConditions := app.termsConditions || interactiveTermsConditions
+
+	var interactivePrivacyPolicy bool
+	if !app.termsConditions {
+		rawAccept := promptInput("Do you accept the Earthly Privacy Policy? [y/N]")
+		if rawAccept == "" {
+			rawAccept = "n"
+		}
+		accept := strings.ToLower(rawAccept)[0]
+
+		interactiveTermsConditions = accept == 'y'
+	}
+	privacyPolicy := app.privacyPolicy || interactivePrivacyPolicy
```

Comment:
I would make this a single prompt to simplify - `Do you accept Earthly's T&Cs <link> and Privacy Policy <link>?` Perhaps even the flag can be a single flag.

Storing separate columns in the DB makes sense though.

Reply:
Yeah? I was just thinking that individual ones would let us re-prompt in the future for just that one. But thats not super common so we can cross that bridge when we get there

Reply:
Also do we want to consolidate to one CLI arg/ env var?

Reply:
I think re-prompting would anyway require some special logic... we might do it on login instead of register. We might just do it over email and that's it. We'll figure it out when we get there.

Reply:
> Also do we want to consolidate to one CLI arg/ env var?

Yeah, I would do that for simplicity.

Reply:
:+1: